systemd/celery.service.d/main.conf

```diff
@@ -0,0 +1,7 @@
+[Service]
+Environment="User=www-data"
```

Comment:
FWIW, I'll take this PR and try to work on improving some of the systemd-isms this week.